solutions/fully-configurable/variables.tf

```diff
@@ -27,17 +27,20 @@ variable "existing_resource_group_name" {
 
 variable "prefix" {
   type        = string
-  nullable    = true
-  description = "Prefix to add to all the resources created by this solution. To not use any prefix value, you can set this value to `null` or an empty string."
+  description = "The prefix to be added to all resources created by this solution. To skip using a prefix, set this value to null or an empty string. The prefix must begin with a lowercase letter and may contain only lowercase letters, digits, and hyphens '-'. It should not exceed 16 characters, must not end with a hyphen('-'), and can not contain consecutive hyphens ('--'). Example: prod-0205-vpc. [Learn more](https://terraform-ibm-modules.github.io/documentation/#/da-implementation-guidelines.md)."
```

Comment:
wrong link - it should link to https://terraform-ibm-modules.github.io/documentation/#/prefix

Please follow the guidelines [here](https://terraform-ibm-modules.github.io/documentation/#/da-implementation-guidelines.md)